batch/batch/worker.py

```diff
@@ -213,7 +218,7 @@ def step(self, name, **kwargs):
             return None
 
         try:
-            c = await docker_call_retry(self.container.show)
+            c = await docker_call_retry(self.container.show, 5)
```

Comment:
I thought you were going to change to 15?

Reply:
I changed start to 15.